WebView.js

```diff
@@ -1,6 +1,9 @@
-import EventEmitter from 'events';
-import React, { Component } from 'react';
-import { WebView as NativeWebView } from 'react-native';
+import EventEmitter from "events";
+import React, { Component } from "react";
+import { WebView as NativeWebView } from "react-native";
+
+// package unique prefix for all messages
+const UNIQUE_MESSAGE_PREFIX = "f251c210-e7c9-42fa-bae3-b9352ec3722a";
```

Comment:
not sure if this identifier should be SO unique 😄
One more point: this identifier should be shared between `react-native` and `WebView` so I'd better move it to separate file. Also `UNIQUE_MESSAGE_PREFIX` is a good name in general, but in that exact case I'd add more context to name to make it obvious that it relates to this package. `RN_MESSAGES_CHANNEL_PREFIX` works fine IMO